lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -117,7 +117,7 @@ def active_span
           end
 
           def add_appsec_tags(processor, trace, span, env)
-            return unless trace
+            return unless trace && span
```

Comment:
wondering if we should pass the scope and check for these from that instead.

same for `add_waf_runtime_tags`